Specs/createScene.js

```diff
@@ -36,6 +36,7 @@ define([
         contextOptions.webgl.antialias = defaultValue(contextOptions.webgl.antialias, false);
 
         var scene = new Scene(options);
+        scene.fog.enabled = false;
```

Comment:
Is this for backwards compatibility with existing tests?  Is this the best approach or should we update those tests?

If we keep this, should we also update Specs/createViewer.js?